articles/tokens/access_token.md

```diff
@@ -1,49 +1,115 @@
 ---
 title: Auth0 access_token
+sitemap: false
 ---
 
 # Auth0 `access_token`
 
 ## Overview
 
-The Auth0 access token, usually called `access_token` in the code samples, is an opaque string which is returned by calls that invoke the Auth0 authentication process and is only used to call the Auth0 API.  The Auth0 access token does not contain any claims, cannot be decoded, does not conform with any standard and is not signed.
+The Auth0 access token, usually called `access_token` in the code samples, is a [JSON Web Token](/jwt) which can be used to authorize users of your APIs when using our [API Authentication](/api-auth)
+
+It conforms to an industry standard (IETF RFC 7519) and contains three parts: A header, a body and a signature. The header contains the type of token and the hash algorithm used on the contents of the token.  
+
+The body, also called the payload, contains identity claims about a user.  There are some non-mandatory claims with registered names, for things like the issuer of the token, the subject of the token (who the claims are about), and the time of issuance.  Any number of additional claims with other names can be added, though care must be taken to keep the JWT within the browser size limitations for URLs.  
+
+The third part of the JWT is the signature which is used by the recipient of a JWT to validate the integrity of the information conveyed in the JWT.
 
 The [Auth0.js documentation](/libraries/auth0js) shows a sample of how to get the `access_token`.
 
+::: warning-banner
+The `access_token` used to be an opaque string that was used to call the Auth0 API, and in some circumstances it still is. For more details on when the `access_token` is a JWT, and when it is an opaque string, please see [Access Token vs ID Token](/tokens/access_token-vs-id_token)
+
+!! Must still do this doc !!
+:::
+
+
 ## How to get Auth0 access token
 
 The Auth0 access token can be obtained in several ways.
 Calls to Lock or library functions that invoke authentication will return the `access_token`.
 
-* Calls to the Lock widget will return `access_token` as shown in the [Lock documentation](/libraries/lock)
-* [Examples using auth0.js](https://github.com/auth0/auth0.js)
-* The [/authorize endpoint in the authentication API](/auth-api) will return `access_token`
+* Calls to the Lock widget will return `access_token` as shown in the [Lock documentation](/libraries/lock) ?? Is this true - can we still use lock ??
```

Comment:
@twistedstream For the purposes of documentation changes, we are documenting the "correct" thing.  Correct in this case is recinding our earlier recommendations that were insecure.  Also, we are finalizing on `/authorize` as the endpoint that will implement this functionality.   Most people should never be aware that `/i/oauth2/authorize` ever existed.

The caveat of this is we may have things (implementation, docs, SDKs) that are not entirely in sync for a while.  This is the consequence of having a hole to dig ourselves out of.